### DISCUSSION TITLE
Igniter problem

### DISCUSSION THREAD

**Discussion by SaharRasoulpour** [firemodels/fds#13435] [General]
Hello, 
I am configuring a wind tunnel setup where I want the igniter to activate at 0.5 seconds, releasing heat at 1500°C towards a bed of pine needles. The igniter is positioned 1.2 cm downstream from the sample and is centered 6 mm from the bottom, targeting the fuel concentration boundary layer. The pine needle bed is located to the right of the igniter. However, when I define non-reacting particles on the igniter, it doesn't propagate enough heat to the pine needle bed. What could be the issue?

!IGNITOR:
&SURF ID='IGN FIRE', TMP_FRONT= 1500 ,RAMP_T='RAMPFIRE', COLOR='RED', SURFACE_VOLUME_RATIO = 8366., LENGTH = 0.07, GEOMETRY = 'CYLINDRICAL' /
&RAMP ID='RAMPFIRE',T= 0.0, F=0.0 /
&RAMP ID='RAMPFIRE',T= 0.5, F=1.0 /
&RAMP ID='RAMPFIRE',T=500., F=0.5 /
&VENT XB = 0.2475,0.255,0.085,0.175,0.015,0.015, SURF_ID='IGN FIRE', IOR=3 /
&PART ID='ignitor particle', SURF_ID='IGN FIRE', STATIC=.TRUE. ,DRAG_COEFFICIENT=  0.1/
&INIT XB=0.2475,0.255,0.085,0.175,0.015,0.02, PART_ID='ignitor particle', N_PARTICLES_PER_CELL=1, CELL_CENTERED=T, DENSITY=1 /

**Comment by drjfloyd** [comment]
If you are predicting fire spread you are doing research. Have you done a
hand calculation to see what heat flux you expect your particles to give?
Have you compared that with the heat flux FDS predicts? Do you know how
close or far you are from igniting the material? Have you tried varying
your particles to see if that has an impact?
